src/styles/index.styl

```diff
@@ -1,4 +1,5 @@
-@import '../../node_modules/cozy-ui/transpiled/react/stylesheet.css'
+@import '~cozy-ui/dist/cozy-ui.min.css'
+@import '~cozy-ui/transpiled/react/stylesheet.css'
```

Comment:
Nit: [The official docs](https://github.com/cozy/cozy-ui#as-a-components-library) advocates to import this file in a jsx file (presumably the bootstraping file), since it contains global variables and nothing else that should be used in the app's css.

Reply:
What is the difference ? 
It seems better to me to have all CSS in one place